bitacross-worker/bitacross/core/bc-musig2-ceremony/src/lib.rs

```diff
@@ -136,10 +136,13 @@ pub struct MuSig2Ceremony<AK: AccessKey<KeyType = SchnorrPair>> {
 	second_round: Option<musig2::SecondRound<SignaturePayload>>,
 	ticks_left: u32,
 	agg_key: Option<PublicKey>,
+	// indicates whether it's check run - signature verification result is returned instead of signature
+	check: bool,
```

Comment:
Does `dry_run` sound better? Or at least `check_run` : ) 

`check` can be very ambiguous